### PR TITLE
Styling

### LINKED ISSUE #7: Prefix the CSS classes
__What should we do?__
Prefix the CSS classes with `camunda-forms` (or similar) to prevent name clashes with other UI libraries

__Why should we do it?__

In the Camunda Platform webapps, we use Bootstrap which also uses the `column` CSS class for layouting with absolute positioning. This results in all form elements displayed ontop of each other:
image

The Platform overrides the CSS from bootstrap for camunda-forms, so it's currently fine for us. It would be nicer to have no name clashes :)

### PR DESCRIPTION
**Pull request by nikku**
This PR accomplishes a few things:

* flatten our CSS classes to avoid collisions (https://github.com/bpmn-io/form-js/pull/2/commits/8242c3e742548c45db1829fdd4e8083811a38dcd)
* adds our form styling
* documents the style usage

---

#### Example Screencapture

capture 012JTc_optimized

--- 

Closes #7

### CODE REVIEW

**Comment by nikku** [comment]
As usual, this can be tried out via `sr`, too:

```
npx @bpmn-io/sr bpmn-io/form-js#styling -c "npm start -- @bpmn-io/form-js"
```

**Comment by andreasgeier** [comment]
It's a perfect match!

As discussed, pls update the color theme to the latest state if possible. Otherwise, your example is already good enough to release.

**Comment by andreasgeier** [comment]
> npx @bpmn-io/sr bpmn-io/form-js#styling -c "npm start -- @bpmn-io/form-js"

❤️  Love this tool - super easy to check the outcome

**Comment by nikku** [comment]
Applied the additional color update. Should be good to merge now.

**Comment by nikku** [comment]
Please provide your actual reviews, otherwise we'll not merge anything :).

**Review by philippfromme — APPROVED** [review]
Looks 💅

File: packages/form-js-viewer/assets/form-js.css [hunk 1]
```diff
@@ -1,76 +1,194 @@
...
+  --color-silver-darken-94: #EFEFEF;
+  --color-silver-darken-80: #CDCDCD;
+  --color-white: #FFFFFF;
+
+  --color-background: var(--color-white);
+  --color-background-disabled: var(--color-silver-darken-94);
+  --color-text: var(--color-gray-darken-23);
+  --color-text-light: var(--color-gray-lighten-56);
+  --color-text-inverted: var(--color-white);
+  --borders: var(--color-grey-lighten-56);
```
Comment (andreasgeier) [inline packages/form-js-viewer/assets/form-js.css:19]: wrong naming for border variable. border styles are not applied.

File: packages/form-js-viewer/assets/form-js.css [hunk 2]
```diff
@@ -1,76 +1,194 @@
...
+  --borders-disabled: var(--color-silver-darken-80);
```
Comment (andreasgeier) [inline packages/form-js-viewer/assets/form-js.css:20]: wrong naming for border-disabled variable. border styles are not applied.

**Review by andreasgeier — COMMENTED** [review]
variable for border-colors is not working

**Comment by nikku** [comment]
Ooops. Fixed via https://github.com/bpmn-io/form-js/commit/e9df755bcb2c5a379906d85b1bb43a4a72b83910.

**Pull request merged**